src/neo-vm/ExecutionEngine.cs

```diff
@@ -206,16 +129,14 @@ private bool ExecuteInstruction()
             if (instruction.OpCode >= OpCode.PUSHBYTES1 && instruction.OpCode <= OpCode.PUSHDATA4)
```

Comment:
Now, we can remove L204  `ExecutionContext context = CurrentContext;`

Reply:
We can't.

https://github.com/neo-project/neo-vm/blob/335a9eff86367ad206b7309ab0b7094448e7d2c6/src/neo-vm/ExecutionEngine.cs#L1183

This `context` is the old context before `CALL`.